modules/azure/aks-global/logs.tf

```diff
@@ -0,0 +1,18 @@
+resource "azurerm_log_analytics_workspace" "log" {
+  name                = "log-${var.environment}-${var.location_short}-${var.name}-${var.unique_suffix}"
+  resource_group_name = data.azurerm_resource_group.log.name
+  location            = data.azurerm_resource_group.log.location
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+}
+
+#tfsec:ignore:azure-storage-queue-services-logging-enabled
+resource "azurerm_storage_account" "log" {
```

Comment:
Do we want to manage these logs in aks-global.
It might be better to move them to core.

My thinking is that if we use core it will become a more natural part of sending all of our logs to.
No matter if it's key vault audit logs or AKS audit logs.

What do you think?